pixie/vm/stdlib.py

```diff
@@ -337,16 +337,36 @@ def apply__args(args):
 @returns(bool)
 @as_var("instance?")
 def _instance(c, o):
-    affirm(isinstance(c, Type), u"c must be a type")
+    from pixie.vm.persistent_vector import PersistentVector
+    if isinstance(c, Type):
+        return true if istypeinstance(o, c) else false
+    elif isinstance(c, PersistentVector):
```

Comment:
Maybe we should introduce an internal `-instance?` (and `-satisfies?`) and then define this version in `stdlib.pxi`?